ui/src/app/sidebar/sidebar.tsx

```diff
@@ -34,6 +34,11 @@ export const Sidebar = (props: SidebarProps) => {
     const locationPath = context.history.location.pathname;
 
     const tooltipProps = {
+        onShow: (instance: any) => {
+            instance.set({
```

Comment:
Is it really not possible to do this in pure css?

Reply:
@jsoref I tried to investigate this again, since this tooltip is using a third-party component, all the styles are controlled by itself, but it can be covered when it is initialized just like the code change in this PR.

But also if you don't want to change any code in javascript, we have another option, we need to add a class name on the container level of the tooltip, and then add media query rules for different screen sizes in css.

Please let me know which way you preferred.

Reply:
In the long term, at least an issue to the 3rd party so that we could eventually fix this with css would be great.

I'm not able to approve, so I can't say whether the project is ok with your current proposal.

If this were my project and you wrote the update ticket+PR, I'd accept this change in the short term.

Reply:
@jsoref Thanks for your comments, either way is good for me, let me ask the owners' thoughts.

Hi @alexec @rbreeze  
Could you please help on this PR? Please find more details below:
The issue the sidebar tooltip width is too large for responsive in small screens(less than 590px), I created the PR to change the tooltip width dynamically when it is initialized, but this needs change the code of javascript.

Another solution is we can add a class name on the container level of the tooltip, and then add media query rules for different screen sizes in css, in this way, we only need to change HTML and CSS.

Could you please leave your comment which way you preferred so this can be merged?

Reply:
I agree with @jsoref that this should be taken care by css. That is the media query way, there are many components which need to be responsive in the UI and it would be best to have a consistent approach for all of them.

Reply:
@saumeya  Thanks for your suggestion, I will create another PR with media query way to fix this issue.